web/themes/contrib/civictheme/civictheme.post_update.php

```
@@ -1317,3 +1318,39 @@ function civictheme_post_update_migrate_logo_image_alt_to_per_type(): Translatab
+  if ($existing) {
+    $editable = \Drupal::configFactory()->getEditable($config_name);
+    unset($config_data['uuid'], $config_data['_core']);
+    $uuid = $editable->get('uuid');
+    if ($uuid) {
+      $config_data['uuid'] = $uuid;
+    }
+    $editable->setData($config_data)->save();
+    return (string) new TranslatableMarkup('Refreshed form display for civictheme_fast_fact_card paragraph from install config.');
+  }
+
+  $config_entity = $storage->createFromStorageRecord($config_data);
```

PHPStan (2.1.54)

[error] Call to an undefined method Drupal\Core\Entity\EntityStorageInterface::createFromStorageRecord().


(method.notFound)